three-js.html

```diff
@@ -1418,3 +1418,48 @@
     });
   </script>
 </dom-module>
+
+<dom-module id="three-js-raycaster">
+  <script>
+    Polymer({
+      is: 'three-js-raycaster',
+      behaviors: [EscherMixins.ReactiveSignal],

```

Comment:
Oh. This will be hard to maintain without special Escher support I'm guessing. Have you tried using this in a general html file somewhere? Something like `test.html`?